frameworks/kafka/tests/test_kerberos_auth.py

```diff
@@ -80,6 +80,7 @@ def kafka_server(kerberos):
                         "hostname": kerberos.get_host(),
                         "port": int(kerberos.get_port())
                     },
+                    "realm": sdk_auth.REALM,
```

Comment:
Using `kerberos.get_realm()` would be better here as this would allow us to better encapsulate these details.